AlfrescoApp/Constants/Constants.m

```diff
@@ -84,6 +84,7 @@
 NSString * const kAlfrescoNodeAddedOnServerContentLocationLocally = @"AlfrescoNodeAddedOnServerContentLocationLocally";
 NSString * const kAlfrescoWorkflowTaskListDidChangeNotification = @"AlfrescoWorkflowTaskListDidChange";
 NSString * const kAlfrescoDocumentEditedNotification = @"AlfrescoDocumentEditedNotification";
+NSString * const kAlfrescoURLHandlingEndedNotification = @"AlfrescoURLHandlingEndedNotification";
```

Comment:
I think this constant and notification name should be renamed. The notification handlers are in MainMenu and RootReveal controllers and it's not at all obvious how the behaviour change (reenable menu item selection) relates to URL handling finishing. The mechanism itself is fine, but I think a better name for the notification might be `kAlfrescoEnableMainMenuAutoItemSelection` as that's the intention of sending the notification. Thoughts?